### DISCUSSION TITLE
Strain Rate ajust in simulation.config

### DISCUSSION THREAD

**Discussion by tmivaniski** [neperfepx/FEPX#54] [Q&A] [answered]
Hello folks,
I'm a new user o FEPX..

I would like to know if there is a possibility to adjust the strain rate according to a function already adjusted from experimental data in the boundary conditions?

## Boundary conditions

boundary _conditions uniaxial_grip

loading_direction Z
loading_face_MAX

Strain_rate (**as a function of plotted data**)

For example, I have linearized a strain rate curve in a micro tensile test, and want to evaluate its response up to a strain of 0.002.
I'd like to check the elastic response as we already know that is a little complicated to take this from the experimental data, even in a miniaturized sample.

Regards.

**Comment by mkasemer** [comment, marked as answer]
No, the current capabilities of FEPX is to apply strain rate jumps at specified load steps. The strain rate is assumed constant over the step, and the equations are only solved at the end of every step.

You could, presumably, create very small load steps and apply a gradual change of the strain rate on each step, such that it would approximate a functional form.
